HighwayPatrol.h

```diff
@@ -2,21 +2,21 @@
 
 #include "Vehicle.h"
 #include "Highway.h"
+#include "Car.h"
+#include "HighwayPatrol.h"
+#include "Motorcycle.h"
+#include "SemiTruck.h"
```

Comment:
You aren't using these classes here.  You're using them in `HighwayPatrol.cpp`
- [ ] move these `#include`s and forward declare instead.